src/layers/domain/core/cpm_product/v1.py

```diff
@@ -4,14 +4,16 @@
 
 from attr import dataclass
 from domain.core.aggregate_root import AggregateRoot
+from domain.core.cpm_system_id.v1 import ProductId
 from domain.core.event import Event
 from domain.core.timestamp import now
-from domain.core.validation import PRODUCT_NAME_REGEX
+from domain.core.validation import CPM_PRODUCT_NAME_REGEX
 from pydantic import BaseModel, Extra, Field
 
 
 class CpmProductIncomingParams(BaseModel, extra=Extra.forbid):
-    product_name: str = Field(regex=PRODUCT_NAME_REGEX, min_length=1)
+    product_team_id: UUID = Field(...)
```

Comment:
Does this work with the createCpmProduct now?

Reply:
yep good point - no need to change this at all

Reply:
also @megan-bower4 maybe be best to add your path parameter model here as `CpmProductPathParams`